packages/components/organisms/f-registration/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.61.1
```

Comment:
NIT – should be a minor bump (as the dependencies are minor version or major version changes)